### PR TITLE
To disable Accessibility support

### PR DESCRIPTION
**Pull request by PeterWangIntel**
This feature is about supporting disabled people, such as reading screen.
So, It's harmless to disable it.

Reduced size = 72K

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@heke123 @halton

File: content/browser/frame_host/render_widget_host_view_child_frame.h [hunk 1]
```diff
@@ -125,6 +125,7 @@ class CONTENT_EXPORT RenderWidgetHostViewChildFrame
   void UnlockCompositingSurface() override;
 #endif  // defined(OS_ANDROID)
 
+#ifndef DISABLE_ACCESSIBILITY
```
Comment (heke123) [inline content/browser/frame_host/render_widget_host_view_child_frame.h]: Lite is only for android now, just leave alone win platform.

**Comment by heke123** [comment]
LGTM

**Comment by PeterWangIntel** [comment]
@halton 
Please help to merge, thx

**Pull request merged**